Okta/OktaAuth/OktaError.swift

```diff
@@ -55,6 +56,8 @@ extension OktaError: LocalizedError {
             return NSLocalizedString("PList name required. See https://github.com/okta/okta-sdk-appauth-ios/#configuration for more information.", comment: "")
         case .pListParseFailure:
             return NSLocalizedString("Unable to read and/or parse. See https://github.com/okta/okta-sdk-appauth-ios/#configuration for more information.", comment: "")
+        case .notConfigured:
+            return NSLocalizedString("You must configure OktaAuth SDK first. See https://github.com/okta/okta-sdk-appauth-ios/#configuration for more information.", comment: "")
```

Comment:
configure OktaAuth -> configure the OktaAuth

Reply:
Fixed.